src/NzbDrone.Core/Indexers/Definitions/AnimeBytes.cs

```diff
@@ -228,10 +228,18 @@ public IList<ReleaseInfo> ParseResponse(IndexerResponse indexerResponse)
                         if (syn.StringArray != null)
                         {
                             synonyms.AddRange(syn.StringArray);
+                            if (syn.StringArray.Count >= 3)
```

Comment:
This new implementation will double add the "splitted" alias because we first add all aliases as is (e.g. if the array is ["this is an example", "this is an example 2, "aliases, comma seperated"] then you add both the complete comma seperated one and the splitted one)

Instead of doing this you should just add index 0 & 1 and then handle everything from 2 onwards with a split by comma.

Reply:
I've replicated the logic from Jackett. There shouldn't be any synonyms from index 3 onwards, according to the AnimeBytes developers.